backend/program/content/overseerr.py

```diff
@@ -32,14 +32,12 @@ def _validate_settings(self):
             return response.ok
         except ConnectTimeout:
             return False
-        # response = json.loads(response.content)
-        # return response['response']
 
     def update_items(self, media_items: MediaItemContainer):
         """Fetch media from overseerr and add them to media_items attribute
         if they are not already there"""
         logger.debug("Getting items...")
-        items = self._get_items_from_overseerr(1000)
+        items = self._get_items_from_overseerr(10000)
```

Comment:
Probably make it from 1 second to 5 seconds (5000) ?

Reply:
This is the amount of items we get from overseerr requests endpoint

Reply:
Oh okay resolved 👍